### PR TITLE
Do backup

### PR DESCRIPTION
**Pull request by swastik-goyal**
This PR is raised to do backup of my previouse codes. 

Signed-off-by: swastik goyal <swastiktanishq1@gmail.com>

### CODE REVIEW

File: conversion/calculator.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+choice = int(input("Enter choice(1/2/3/4):"))
+
+num1 = int(input("Enter first number: "))
+num2 = int(input("Enter second number: "))
+
+#print("s.show")
+#print("h.hide")
+#print("invalid.show")
+
+#choise2 = input("formula or not: ")
```
Comment (pankajgoyal2) [inline conversion/calculator.py]: correct spelling of choise!

File: conversion/calculator.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+print("4.Divide")
+
+choice = int(input("Enter choice(1/2/3/4):"))
+
+num1 = int(input("Enter first number: "))
+num2 = int(input("Enter second number: "))
+
+#print("s.show")
+#print("h.hide")
+#print("invalid.show")
```
Comment (pankajgoyal2) [inline conversion/calculator.py]: what are these comments about?

File: conversion/tempurator-convertor [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+print("""
+Units of temperature are : 
+
+1. celsius
+2. fahrenheit
+3. kelvin
+4. rankine
+5. delisle
+6. newton""")
+
```
Comment (pankajgoyal2) [inline conversion/tempurator-convertor]: print above lines in readable format.

File: conversion/tempurator-convertor [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+# use converter
+
+a = input("To use converter press 'c' OR to close the program press any key   key = ")
```
Comment (pankajgoyal2) [inline conversion/tempurator-convertor]: you can add the helper function in this program

**Review by pankajgoyal2 — CHANGES_REQUESTED** [review]

File: conversion/tempurator-convertor (hunk 4 above)
Comment (swastik-goyal) [reply conversion/tempurator-convertor]: I have made some helpful changes in the code. But please tell me what is helper function.

**Pull request merged**